package.json

```diff
@@ -47,6 +47,7 @@
   },
   "workspaces": [
     "example",
+    "example-wasm",
```

Comment:
Good idea! Perhaps we should do the move of `example` (React Native) in a separate PR to `main` instead, then have this branch just move the current app (to keep the "Files changed" more relevant). If we start working on the upcoming examples before the emscripten branch is merged into `main`, it could be good to make that separation there as well.

Reply:
Done ✅  Moved it into `examples/` but haven't renamed the directory yet. I also think any of your suggested names would work.